data/westminster/wlc.yaml

```diff
@@ -4976,7 +4976,7 @@ questions:
     number: 160
     answer: >-
       It is required of those that hear the word preached, that they attend
-      upon it with diligence,[1] preparation,[2] and prayer;[3] examine:
+      upon it with diligence,[1] preparation,[2] and prayer;[3] examine
       What they hear by the scriptures;[4] receive the truth with faith,[5]
```

Comment:
Would you mind changing the uppercase `W` in `What` on the following line to a lowercase one?